.rubocop.yml

```diff
@@ -0,0 +1,54 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
inherit a rubocop file is the same as overwriting the basic config. Based on the instructions provided on this [link](https://github.com/microverseinc/linters-config/tree/master/ruby) you should not change the config file.
![image](https://user-images.githubusercontent.com/38625956/79899612-d9766b00-842e-11ea-8fe4-7bda147d5d00.png)